shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/conf/CircusTrainTableReplications.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (C) 2016-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.replicator.exec.conf;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import com.hotels.bdp.circustrain.api.conf.TableReplications;
+
+@Configuration
+@ConfigurationProperties(prefix = "table-replications")
+public class CircusTrainTableReplications extends TableReplications {}
```

Comment:
If you run this do we not get errors because of the constraints e.g. `@NonNull`, `@TableReplicationFullReplicationModeConstraint` or do we not validate the config in SY?

Reply:
We do get errors and that is why CT conf classes need to be copied over to SY. It all looks ugly now :(